packages/frontend/src/pages/TokenSwap/ui/Success.js

```diff
@@ -5,78 +5,100 @@ import styled from 'styled-components';
 import FormButton from '../../../components/common/FormButton';
 import SafeTranslate from '../../../components/SafeTranslate';
 import AvatarSuccessIcon from '../../../components/svg/AvatarSuccessIcon';
+import FailIcon from '../../../components/svg/FailIcon';
+import { toSignificantDecimals } from '../../../utils/amounts';
+import { useSwapData } from '../model/Swap';
+import { DECIMALS_TO_SAFE } from '../utils/constants';
 
 const StyledContainer = styled.div`
     > svg {
         margin: 30px auto;
         display: block;
     }
+`;
 
-    div.buttons-bottom-buttons {
-        margin-top: 55px;
+const Header = styled.div`
```

Comment:
it's hard to understand from jsx markup is it styled component or react jsx class. Better to make this names more obvious like StyledHeader or smth like this.
Not a blocker though